src/addon-manager.js

```diff
@@ -594,15 +594,15 @@ class AddonManager extends EventEmitter {
     await this.loadAddon(packageId);
   }
 
-  async disableAddon(packageId) {
+  async disableAddon(packageId, wait = false) {
```

Comment:
Why default to false? There is only one place which this is being called, and thats below where we pass in true.

Reply:
Just to retain compatibility with the other function, in case we decide to use disableAddon elsewhere.